src/yarpmotorgui-qt/partitem.cpp

```diff
@@ -537,7 +536,7 @@ void PartItem::onSendPid(int jointIndex,Pid newPid)
 }
 void PartItem::onCalibClicked(JointItem *joint)
 {
-    const int jointIndex = joint->getJointIndex();
+    joint->getJointIndex();
```

Comment:
I am fine with Q_UNUSED(joint) if this makes you sleep better.

The author of that code should also have a look, remove the function call and test it. If the author does not have time we can live with this.